docs/develop/go/temporal-nexus.mdx

```diff
@@ -0,0 +1,476 @@
+slug: /nexusgosdk
+toc_max_heading_level: 4
+keywords:
+  - temporal nexus go sdk
+
+tags:
+  - temporal-nexus
+  - go-sdk
+  - nexus-endpoints
+  - nexus-operations
+  - nexus-service-contract
+---
+
+Use Temporal Nexus to connect durable executions within and across Namespaces using a Nexus Endpoint, a Nexus Service contract, and Nexus Operations.
+
+This page shows how to do the following:
+
+- [Run a development Temporal Service with Nexus enabled](#run-the-temporal-nexus-development-server)
+- [Create caller and handler Namespaces](#create-caller-handler-namespaces)
+- [Create a Nexus Endpoint to route requests from caller to handler](#create-nexus-endpoint)
+- [Define the Nexus Service API contract](#define-nexus-service-api-contract)
+- [Develop a Nexus Service and Operation handlers](#develop-nexus-service-operation-handlers)
+- [Develop a caller Workflow that uses a Nexus Service](#develop-caller-workflow-nexus-service)
+- [Make Nexus calls across Namespaces with a development Server](#nexus-calls-across-namespaces-dev-server)
+- [Make Nexus calls across Namespaces in Temporal Cloud](#nexus-calls-across-namespaces-temporal-cloud)
+
+## Run a development server with Nexus enabled {#run-the-temporal-nexus-development-server}
+
+The first step in working with Temporal Nexus involves starting a Temporal server with Nexus enabled.
+
+Prerequisites:
+
+- [Install the latest Temporal CLI](https://docs.temporal.io/develop/go/core-application#run-a-development-server)
+- [Install the latest Temporal Go SDK](https://docs.temporal.io/develop/go/core-application#install-a-temporal-sdk) (v1.29.1 or higher)
+
+### Start the Temporal Development Server {#start-the-temporal-development-server}
+
+Start the Temporal Development Server by using the server start-dev command, with `system.enableNexus=true`.
+The HTTP port is required for Nexus communications.
+
+```
+temporal server start-dev --http-port 7243 --dynamic-config-value system.enableNexus=true
+```
+
+This command automatically starts the Web UI, creates the default Namespace, and uses an in-memory database.
+
+The Temporal Server should be available on `localhost:7233` and the Temporal Web UI should be accessible at [http://localhost:8233](http://localhost:8233).
+
+## Create caller and handler Namespaces {#create-caller-handler-namespaces}
+
+Before setting up Nexus endpoints, create separate namespaces for the caller and handler.
+
+```
+temporal operator namespace create --namespace my-target-namespace
+temporal operator namespace create --namespace my-caller-namespace
+```
+
+## Create a Nexus Endpoint to route requests from caller to handler {#create-nexus-endpoint}
+
+After establishing caller and handler Namespaces, the next step is to create a Nexus Endpoint to route requests.
+
+```go
+temporal operator nexus endpoint create \
```

Comment:
The endpoint name is wrong. I would stick closer to the sample's README and run these steps locally to verify they're correct.

https://github.com/temporalio/samples-go/tree/main/nexus#create-nexus-endpoint